dotty-docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM ubuntu:19.10
 
 # The default locale is "POSIX" which is just ASCII.
 ENV LANG C.UTF-8
```

Comment:
the sbt version below is 1.3.6, might as well bump it to 1.3.10 to get the latest launcher